SS_readstarter.tpl

```diff
@@ -640,20 +640,18 @@
   int depletion_basis;
   int depletion_multi;
   int depletion_log;
-  init_int depletion_basis_rd; // 0=skip; 1=B0; 2=Bmsy; 3=B_styr; 4=B_endyr; 5=dynamic_Bzero; values >=11 invoke multiyr with 10's digit; >=100 invoke log(ratio) with hundreds digit
+  init_number depletion_basis_rd; // 0=skip; 1=B0; 2=Bmsy; 3=B_styr; 4=B_endyr; 5=dynamic_Bzero; values >=11 invoke multiyr with 10's digit; append .1 to invoke log(ratio) with hundreds digit
```

Comment:
Can we add an example here of what that would look like for a specific option?

Reply:
Alternatively, we could include that in the manual if it that would be preferred.